### PR TITLE
Upgraded rails from 5.2.4.5 to 6.0.3.5

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a developer, I need to upgrade GIDS to Rails v.6.0.3.5 so that GIDS is using the most recent version of the library.
## Original issue(s)

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/20067

Vets-api linked PR: https://github.com/department-of-veterans-affairs/vets-api/pull/4069

Ruby on Rails 6.0 release notes: https://guides.rubyonrails.org/6_0_release_notes.html 
## Testing done
local smoke test

## Acceptance criteria
- [x] Rails is on version 6.0.3.5.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: spec/models/institution_builder_spec.rb [hunk 1]
```diff
@@ -462,7 +462,7 @@
       let(:ipeds_ic_py) { IpedsIcPy.first }
       let(:ipeds_ic_ay) { IpedsIcAy.first }
 
-      let(:nil_ipeds_ic_ay) { IpedsIcPy::COLS_USED_IN_INSTITUTION.each_with_object({}) { |v, o| o[v] = nil } }
+      let(:nil_ipeds_ic_ay) { IpedsIcPy::COLS_USED_IN_INSTITUTION.index_with({}) { |v, o| o[v] = nil } }
```
Comment (LindseySaari) [inline spec/models/institution_builder_spec.rb:465]: Can this be moved into a separate PR

Comment (ksantiagomorgan6) [reply spec/models/institution_builder_spec.rb:465]: It won't pass continuous-integration/jenkins/branch without it

**Review by LindseySaari — APPROVED** [review]
